lib/class-wp-rest-site-controller.php

```diff
@@ -116,11 +116,25 @@ public function get_item_schema() {
 					'format'      => 'uri',
 					'context'     => array( 'view', 'edit' ),
 				),
+				'admin_email' => array(
+					'description' => __( 'Email Address' ),
+					'type'        => 'string',
+					'format'      => 'email',
+					'context'     => array( 'view', 'edit' ),
```

Comment:
@danielbachhuber OK for some reason I was thinking that we would expose these in GET requests too if the user was authenticated with the proper capability.